dis_snek/models/discord/channel.py

```diff
@@ -1998,8 +2034,10 @@ async def close_stage(self, reason: Absent[Optional[str]] = MISSING) -> None:
 
 @define()
 class GuildForum(GuildChannel):
-    available_tags: list[ThreadTag] = field(factory=list)
+    available_tags: List[ThreadTag] = field(factory=list)
     """A list of tags available to assign to threads"""
+    last_message_id: Optional[Snowflake_Type] = field(default=None)
+    # TODO: Implement "template" once the API supports them
```

Comment:
The API already supports templates. You send it as part of the JSON payload in the form of a string as a `PATCH` request to the `Modify Channel` endpoint. The lacking support so far is to make it as a `POST` request upon creation.

Reply:
What does a template look like?